rpc-server/src/config.rs

```diff
@@ -1,14 +1,12 @@
 use crate::modules::blocks::CacheBlock;
-use clap::Parser;
-
-pub const DEFAULT_RETRY_COUNT: u8 = 3;
+use clap::{Parser, Subcommand};
 
 #[derive(Parser)]
 #[clap(author, version, about, long_about = None)]
 pub struct Opts {
-    // near network rpc url
-    #[clap(long, env = "NEAR_RPC_URL")]
-    pub rpc_url: http::Uri,
+    // Chain ID: testnet or mainnet
+    #[clap(subcommand)]
+    pub chain_id: ChainId,
```

Comment:
In `LakeConfigBuilder` both `mainnet` and `testnet` are shortcuts to set the proper `s3_bucket_name`. Since we already has it as parameter for ReadRPC, I am sure we don't need `chain_id` in ReadRPC config. Please refactor it accordingly